packages/lib/response/cache.ts

```diff
@@ -0,0 +1,50 @@
+import { revalidateTag } from "next/cache";
+
+interface RevalidateResponses {
```

Comment:
Can we rename these to `revalidateProps` to stick more with current conventions?

Reply:
btw why should be call it `revalidateProps` why not `revalidateArgs`? `props` is more of a react terminology.

Reply:
I would like to stick with one for the Formbricks project :-) 
We could also rename them in the whole project, but for that I would need a pretty cool alternative ;-) 
For me this is still one big react project

Reply:
Hmm, okay.